test/e2e/suites/import-gitops/import_gitops_test.go

```diff
@@ -32,41 +32,6 @@ import (
 	"github.com/rancher/turtles/test/e2e/specs"
 )
 
-var _ = Describe("[Docker] [Kubeadm] - [management.cattle.io/v3] Migrate v1 to management v3 cluster functionality should work", Label(e2e.ShortTestLabel), func() {
-	BeforeEach(func() {
-		komega.SetClient(setupClusterResult.BootstrapClusterProxy.GetClient())
```

Comment:
can you remove komega package import from the import list if not needed?

Reply:
Yep, that caused a failure. Will push and try again.

Reply:
Should be passing now 🙏🏻